eZ/Publish/Core/Persistence/Legacy/Tests/Content/_fixtures/extract_content_from_rows.php

```diff
@@ -21,15 +21,15 @@
     'ezcontentobject_version_creator_id' => '14',
     'ezcontentobject_version_created' => '1313061317',
     'ezcontentobject_version_status' => '1',
-    'ezcontentobject_version_contentobject_id' => '226',
+    //'ezcontentobject_version_contentobject_id' => '226',
```

Comment:
Why do you need to comment it out? Is this to be removed?

Reply:
yes, not needed anymore. I guess it will fail tests if it is still here.